src/org/omegat/gui/dialogs/VersionCheckDialog.java

```diff
@@ -61,8 +61,11 @@ protected void done() {
                         new VersionCheckDialog(VersionChecker.getInstance().getRemoteVersion()).show(parent);
                     }
                 } catch (Exception e) {
-                    Log.log(e);
-                    JOptionPane.showMessageDialog(parent, e.getLocalizedMessage(), OStrings.getString("ERROR_TITLE"),
+                    Log.logWarningRB("VERSION_CHECK_FAILED");
+                    Log.log(e.toString());
```

Comment:
You can combine these two lines as well if you use a placeholder in the properties string.